aries-backchannels/aries-vcx/Cargo.toml

```diff
@@ -22,4 +22,4 @@ pickledb = "0.4.1"
 uuid = { version = "0.8.2", features = ["serde", "v4"] }
 futures-util = "0.3.15"
 futures = "0.3.15"
-aries-vcx = { git = "https://github.com/hyperledger/aries-vcx", rev = "a4fe9fb" }
+aries-vcx = { git = "https://github.com/hyperledger/aries-vcx", rev = "0e47a081f" }
```

Comment:
This revision isn't working (commit not in aries-vcx repo)
I'm testing with `edc78bd15355ce1efc9dd338fd3a6cd4f93ad287`

Reply:
Updated the aries-vcx revision, make sure you have commit `e86e46d62adb41b29facce5d9bf25894da63f7c3` of this branch (`mirgee:feature/aries-vcx-backchannel`) checked out.